backend/variants/models/maintenance.py

```
@@ -109,3 +109,80 @@ def refresh_variants_smallvariantsummary():
+    FROM variants_smallvariant AS variants
+    WHERE NOT EXISTS (SELECT 1 from excluded_case_ids AS e WHERE e.case_id = variants.case_id)
+) AS variants_per_case
+GROUP BY (release, chromosome, start, "end", bin, reference, alternative)
+"""
+
+
+def create_variants_smallvariantsummary():
+    """Create the ``SmallVariantSummary`` materialized view."""
+
+    with transaction.atomic():
+        with connection.cursor() as cursor:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)